docs/content/contributing/bicep/bicep-contribution-flow/owner-contribution-flow.md

```diff
@@ -0,0 +1,180 @@
+
+{{< hint type=info >}}
+
+GitHub will use the following email addresses to Cc you if you're subscribed to a conversation. The second Cc email address matches the notification reason.
+
+{{< /hint >}}
+
+| Type of Notification | GitHub Email Address | Notification Reason |
+|-|-|-|
+| @Mentions | mention@noreply.github.com | You were mentioned on an issue or pull request. |
+| @Team Mention | team_mention@noreply.github.com | A team you belong to was mentioned on an issue or pull request |
+| Subscribed | subscribed@noreply.github.com | There was an update in a repository you're watching. |
+| Assign | assign@noreply.github.com | You were assigned to an issue or pull request. |
+| Comment | comment@noreply.github.com | You commented on an issue or pull request. |
+
+For a full list of GitHub notification types, see [Filtering Email Notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#filtering-email-notifications).
+
+---
+
+### 5. Contribution Checklist
+
+This checklist can be used by anyone (author/contributor/owner) developing AVM Bicep Modules.
+
+1. Before begining any work a new module a valid [Issue: New AVM Module Proposal](https://github.com/Azure/Azure-Verified-Modules/issues/new?assignees=&labels=Type%3A+New+Module+Proposal+%3Abulb%3A%2CNeeds%3A+Triage+%3Amag%3A&projects=Azure%2F529&template=module_proposal.yml&title=%5BModule+Proposal%5D%3A+%60MODULE_NAME_BETWEEN_BACKTICKS%60) needs to be created. Instructions for creating the module proposal are outlined in the issue template. Pay particular attention to the questions and associated links to fill out the proposal acurately. Please do not start work on your proposed module until you receive a notification that your proposal has been accepted.
+
+2. Fork bicep-registry-modules [BRM](https://github.com/Azure/bicep-registry-modules), if you use an existing fork, ensure it's up to date with origin/BRM.
+
+- Make sure all workflows are [disabled by default](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/enable-or-disable-workflows/) once you forked the BRM repo, to prevent any accidental deployments into your Azure test environment resulted by an automated deployment.
+
+3. Create a new branch from your forked repository to develop your module.
+
+4. If you created a new module you have to create its corresponding workflow file (see [here](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/#4-implement-your-contribution)).
+
+- In order to run your e2e tests in your fork, this workflow file has to be put into `main` first so it can be run against your feature branch (GitHub Workflow appear by default and can run on feature branches only when they are present in main).
```

Comment:
```suggestion
    - In order to run your e2e tests in your fork, this workflow file has to be put into `main` first so it can be run against your feature branch (GitHub Workflow appear by default and can run on feature branches only when they are present in main).
```